cspell/.cspell.json

```diff
@@ -1039,6 +1040,7 @@
         "xorriso",
         "xosc",
         "XYZI",
+        "XYZIRADT",
```

Comment:
Question: What does `XYZIRADT` stand for? I'd like to add the meaning of the word to the commit message, like #87.

Reply:
`XYZIRADT` is the message type, similar to `XYZI`.
It stands for X, Y, Z, Intensity, Ring, Azimuth, Distance and Timesramp.

Reply:
I`ll add it to PR comment.